### ISSUE TITLE
Segmentation fault

### ISSUE THREAD

**Issue by MyungHaSong** [j96w/DenseFusion#26]
Hi, Thanks for sharing the code!!!

I ran download.sh file and success 

so I run 

sh .experiments/scripts/train_linemod.sh

after object buffer loaded, I get this error 

>>>>>>>>----------Dataset loaded!---------<<<<<<<<
length of the training set: 2373
length of the testing set: 1336
number of sample points on mesh: 500
symmetry object list: [7, 8]
2019-04-08 03:59:01,660 : Train time 00h 00m 00s, Training started
/home/user/miniconda/envs/py36/lib/python3.6/site-packages/torch/nn/functional.py:1749: UserWarning: Default upsampling behavior when mode=bilinear is changed to align_corners=False since 0.4.0. Please specify align_corners=True if the old behavior is desired. See the documentation of nn.Upsample for details.
  "See the documentation of nn.Upsample for details.".format(mode))
/home/user/miniconda/envs/py36/lib/python3.6/site-packages/torch/nn/modules/container.py:91: UserWarning: Implicit dimension choice for log_softmax has been deprecated. Change the call to include dim=X as an argument.
  input = module(input)
Segmentation fault (core dumped)

How can I solve this error and run train code

**Comment by j96w** [comment]
Hi, sorry I think I just can't debug with a "Segmentation fault". Any more information?

**Comment by gouxiangchen** [comment]
Hi，I get something wrong in download.sh. Can you share the dataset you download :) Thanks a lot!

**Comment by BarbeBleue** [comment]
@gouxiangchen Without any other information I can only advise you to copy directly the lines from download.sh into your terminal or just open the links in your browser:
[YCB-Video](https://docs.google.com/uc?export=download&id=1if4VoEXNx9W3XCn0Y7Fp15B4GpcYbyYi)
[Linemod](https://docs.google.com/uc?export=download&id=1YFUra533pxS_IHsb9tB87lLoxbcHYXt8)
[Trained Checkpoints](https://docs.google.com/uc?export=download&id=1bQ9H-fyZplQoNt1qRwdIUX5_3_1pj6US)

**Comment by gouxiangchen** [comment]
@BarbeBleue  Thanks for your advice! It turns out to be my network issue, and it works fine now ~.~

**Comment by MyungHaSong** [comment]
@j96w 
thank you for reply. but I solved this problem.. 
it may be dataset path error

**Comment by RobbyHuang** [comment]
@MyungHaSong ,hi ,I met the same problem.  my datasets path is './datasets/ycb/YCB_Video_Dataset'. Do you know how to deal with it?
